src/components/Form.js

```diff
@@ -50,11 +50,9 @@ class Form extends React.Component {
     const { name, age, sex, email } = this.props.fields.toJS();
     return (
       <Page>
-        <h2 className="large-header">MYPCO Lite Score Generator</h2>
+        <h2 className="large-header">MYPCo. Lite Score Generator</h2>
         <p className="body form-intro-text">
-          It is a long established fact that a reader will be distracted by the
-          readable content of a page when looking at its layout. The point of
-          using Lorem Ipsum is that it has a more-or-less normal distribution
+          Welcome to MYP Boost! YOU have made a commitment to YOURSELF and to those who are close to YOU. The key to MYP and YOUR success is the application of small consistent steps applied daily to YOUR routine.
```

Comment:
I'd break the text here and in other places into separate lines, just to increase readability. Roughly the length it was before would be swaz.